chapters/use-a-database/connect-to-a-database.md

```diff
@@ -1,6 +1,6 @@
 ## Connect to a database
 
-There are a few things you need to use Entity Framework Core to connect to a database. Since you used `dotnet new` and the MVC + Individual Auth template to set your project, you've already got them:
+There are a few things you need, to use Entity Framework Core to connect to a database. Since you used `dotnet new` and the MVC + Individual Auth template to set your project, you've already got them:
```

Comment:
No comma needed here!

Reply:
I have no idea what went through my head at the time of writing.